config/global.php

```diff
@@ -72,4 +72,16 @@
                 'tld' => false,
             ))));
     },
+
+    'tracker.request.processors' => array(),
```

Comment:
is it possible to add a little comment here what it is? Or maybe it is obvious then forget it